### PR TITLE
fix(tabs): unset z index

### PR DESCRIPTION
**Pull request by alexandereneroth**
z index were unnecessarily high.

NOTE: Might technically be a breaking change. However, I believe users getting negatively affected would be very rare. As such, I think leaving it will cause far more issues for users than removing it (z-index).

**How to test:**
Open the deploy preview and check that:
- tabs have no z-index set
- scroll buttons still appear and work as expected
- check for other issues

### CODE REVIEW

**Comment by timrombergjakobsson** [comment]
Did we discuss this one in a checkin?

**Comment by alexandereneroth** [comment]
@mJarsater What was your reasoning for keeping z-index on tabs? I remember we discussed this.
I think they could be removed. Technically it might be a breaking change - but I believe the benefit of changing it outweighing the drawback of leaving it.

**Comment by mJarsater** [comment]
> as your reasoning for keeping z-index on tabs? I remember we discussed this.
> I think they could be removed. Technically it might be a breaking change - but I believe the benefit of changing it outweighing the drawback of leaving it

I think the only reason was to be sure that the previous/next buttons are showing correctly.

**Comment by alexandereneroth** [comment]
> > as your reasoning for keeping z-index on tabs? I remember we discussed this.
> > I think they could be removed. Technically it might be a breaking change - but I believe the benefit of changing it outweighing the drawback of leaving it
> 
> I think the only reason was to be sure that the previous/next buttons are showing correctly.

Alright, then I think making sure of that can be part of testing this PR.

**Comment by alexandereneroth** [comment]
> https://pr-376.d3fazya28914g3.amplifyapp.com

Don't think so, anyway I think it would not be worth the time

**Comment by alexandereneroth** [comment]
> Did we discuss this one in a checkin?

Don't think so, anyway I think it would not be worth the time

**Review by mJarsater — APPROVED** [review]

**Comment by TatyanaBumazhkina** [comment]
Checked deploy preview page for Inline Tabs and Navigation Tabs. Elements don't have z-index in HTML code.

File: packages/core/src/components/tabs/inline-tabs/inline-tabs.scss
Comment (nathalielindqvist) [inline packages/core/src/components/tabs/inline-tabs/inline-tabs.scss:37]: Asking because I don't know - will the set z-index not lead to the scroll buttons ending up behind something else if the user has something in the surrounding context that has a higher z-index?

**Comment by timrombergjakobsson** [comment]
> > Did we discuss this one in a checkin?
> 
> Don't think so, anyway I think it would not be worth the time

?

**Review by nathalielindqvist — APPROVED** [review]
Looks good! Just a small comment regarding context 😄

**Review by theJohnnyMe — APPROVED** [review]
Checked in Storybook, looks OK. I would say please test this in one of demo pages too by packing the code, just to see how it interacts with other components.

**Comment by alexandereneroth** [comment]
> > > Did we discuss this one in a checkin?
> > 
> > 
> > Don't think so, anyway I think it would not be worth the time
> 
> ?

Sorry, I misread. Thought you wrote "Should we discuss this one in a checkin?"

I think I might have brought it up, can't quite remember.

**Comment by alexandereneroth** [comment]
> Checked in Storybook, looks OK. I would say please test this in one of demo pages too by packing the code, just to see how it interacts with other components.

I suggest we can write a Playwright test for it instead, what do you think?
Packing it for the demo pages would be a bit of work now as they use the wrappers.

**Comment by timrombergjakobsson** [comment]
> > Checked in Storybook, looks OK. I would say please test this in one of demo pages too by packing the code, just to see how it interacts with other components.
> 
> I suggest we can write a Playwright test for it instead, what do you think? Packing it for the demo pages would be a bit of work now as they use the wrappers.

Sounds like a good idea! On the demo pages note though, we can still pack it right? I tried earlier, and it worked.

**Comment by alexandereneroth** [comment]
> > > Checked in Storybook, looks OK. I would say please test this in one of demo pages too by packing the code, just to see how it interacts with other components.
> > 
> > 
> > I suggest we can write a Playwright test for it instead, what do you think? Packing it for the demo pages would be a bit of work now as they use the wrappers.
> 
> Sounds like a good idea! On the demo pages note though, we can still pack it right? I tried earlier, and it worked.

I guess. I feel quite confident with this one though. Leave a review comment or reject the PR if you think it should be tested like that before merging. 
That green squash and merge button looks awfully tempting.

**Comment by timrombergjakobsson** [comment]
> > > > Checked in Storybook, looks OK. I would say please test this in one of demo pages too by packing the code, just to see how it interacts with other components.
> > > 
> > > 
> > > I suggest we can write a Playwright test for it instead, what do you think? Packing it for the demo pages would be a bit of work now as they use the wrappers.
> > 
> > 
> > Sounds like a good idea! On the demo pages note though, we can still pack it right? I tried earlier, and it worked.
> 
> I guess. I feel quite confident with this one though. Leave a review comment or reject the PR if you think it should be tested like that before merging. That green squash and merge button looks awfully tempting.

You already have 3 approved reviews, so go ahead ;)

**Pull request merged**
